src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -568,6 +632,7 @@ private IEnumerable<AssociativeNode> GetDeltaAstListModified(List<Subtree> modif
         {
             var deltaAstList = new List<AssociativeNode>();
             csData.RemovedBinaryNodesFromModification = new List<AssociativeNode>();
+            csData.ModifiedNodesForRuntimeSetValue = new List<AssociativeNode>();
```

Comment:
You don't need to initialize this `List` here if it is passed as an `out` parameter to `GetModifiedNodes`